### PR TITLE
fix(awful: hotkeys): fix sorting of hotkeys

### PR DESCRIPTION
**Pull request by actionless**

### CODE REVIEW

**Comment by Elv13** [comment]
+1

File: lib/awful/hotkeys_popup/widget.lua [hunk 1]
```diff
@@ -262,10 +257,19 @@ local function create_wibox(s, available_groups)
...
+                local length = string.len(key.key or '') + string.len(key.description or '')
+                local modifiers = key.mod
+                if not modifiers or modifiers == "none" then
+                    modifiers = ""
+                else
+                    length = length + string.len(modifiers) + 1 -- +1 for "+" character
+                    modifiers = markup.fg(widget.modifiers_color, modifiers.."+")
+                end
+                local rendered_hotkey = markup.font(widget.title_font,
+                    modifiers .. (key.key or "") .. " "
```
Comment (Elv13) [inline lib/awful/hotkeys_popup/widget.lua]: The indentation here is a little misleading.

Comment (actionless) [inline lib/awful/hotkeys_popup/widget.lua]: any suggestions?

Comment (Elv13) [inline lib/awful/hotkeys_popup/widget.lua]: NVM, It isn't that misleading after all

**Comment by psychon** [comment]
:+1: from me, too

**Pull request merged**